pest/examples/parens.rs

```diff
@@ -0,0 +1,105 @@
+extern crate pest;
```

Comment:
In particular, I'm worried about the deeply nested functions. I wouldn't wish debugging that on anyone. If it's auto generated then it's probably fine because we'll be the only ones having to debug the generated output.